include/algebra_vector.h

```diff
@@ -50,6 +50,10 @@ OSQPVectorf* OSQPVectorf_view(const OSQPVectorf *a,
                               c_int              head,
                               c_int              length);
 
+/* Points existing subview somewhere else.  (Does not use MALLOC)
+ */
+void OSQPVectorf_view_update(OSQPVectorf *a, const OSQPVectorf *b, c_int head, c_int length);
+
```

Comment:
Since this function is currently only used by MKL-CG, we will not need it here once we move the linsys solvers to the `algebra` directory.

Reply:
The default algebra has a parallel function, and its being used too:
https://github.com/osqp/osqp/blob/4e7787e3a13db5fdecee7e4f0593e64d606bfedc/algebra/default/vector.c#L144

Maybe we tackle this as a separate issue?

Reply:
Ok, I will tackle this and similar issues in a separate PR.